index.rst

```diff
@@ -1,18 +1,16 @@
 .. _index:
 
 ==========================================
-Normative specification for CellML 2.0
+CellML 2.0 Specification
 ==========================================
```

Comment:
I would remove some "=" to make it the same length as the header.

Reply:
Same below.

Reply:
or leave it...I was planning to put the normative spec bit back in once things settle down ;-)

Reply:
Will do elsewhere throughout later on